pkg/api/v1/routes/handlers.go

```diff
@@ -79,8 +79,54 @@ func (r *Routes) serverConditionCreate(c *gin.Context) (int, *v1types.ServerResp
 		}
 	}
 
+	return r.conditionCreate(otelCtx, c, kind, serverID)
```

Comment:
Note: the `inventory` condition needs to be created with certain parameters - you can follow the same pattern as in mctl - https://github.com/metal-toolbox/mctl/blob/fd42322cb93842dc993dee44d70f986230b9e6c0/cmd/collect/inventory.go#L46

Reply:
Thanks! Do you mean I can reuse exactly the same pattern for new added `add server`? Since we are reusing the `inventory` kind